### DISCUSSION TITLE
Alert Not Showing on Wazuh Dashboard Despite Decoder and Rule Working Correctly

### DISCUSSION THREAD

**Discussion by karimbeltesing** [wazuh/wazuh#26992] [Q&A] [answered]
Hello everyone,

I have written a custom decoder and rule for a specific syslog log in my Wazuh setup, and it works as expected:

When I run `tail -f /var/ossec/logs/archives/archives.log`, I can see the syslog log arriving at my server.
Using `/var/ossec/bin/wazuh-logtest`, I can confirm that:

- The log is decoded correctly by my custom decoder.
- My custom rule is triggered, and the system indicates that an alert should be generated.

However, when I check the Wazuh dashboard, the alert doesn't appear.

Thank you for your help

**Comment by karimbeltesing** [comment]
After troubleshooting:
- I find the alerts sent to /var/ossec/logs/alerts/alerts.json but not shown in the dashboard.
- There is enough desk space in all of the wazuh components.

**Comment by SirStephanikus** [comment]
Hello Karim

Archive != Alert

If the alert is written to your alerts.log file, then is should 100% shown in your dashboard. Perhaps your filter is set wrong?
Who receives the syslog?

**Reply by karimbeltesing** [reply, marked as answer]
Hello @SirStephanikus 
I resolved the issue yesterday. I came across someone on Reddit who had a similar problem, and they provided a solution that worked for me.

Here is the post : 
https://www.reddit.com/r/Wazuh/comments/1d8vq4u/alerts_hitting_alertjson_but_not_the_dashboard/

**Reply by SirStephanikus** [reply]
Thx for the update, and interesting post from reddit. I'll bookmark it.

**Reply by karimbeltesing** [reply]
Glad you found it interesting, and thank you for your help and time.

**Comment by SLMAN-BLK** [comment]
i have a probleme thats look like yours but for me i don't find the log on 
knowing thats i test the log /var/ossec/bin/wazuh-logtest
and my rule say alert should be generated and when i check tail -n 5 /var/ossec/logs/alerts/alerts.json
i dont find the alert
